commands.js

```diff
@@ -228,17 +228,27 @@ const operations = {
     },
 
     /* Publish pipeline template */
-    publishPipelineTemplate: {
+    publish_pipeline_template: {
         opts: {
             json: { abbr: 'j', flag: true, help: 'Output result as json' }
```

Comment:
do you need to add https://github.com/screwdriver-cd/template-main/blob/master/commands.js#L12
```suggestion
            json: { abbr: 'j', flag: true, help: 'Output result as json' },
            tag: { abbr: 't', default: 'latest', help: 'Add template tag' }
```